src/parser.rs

```diff
@@ -202,6 +203,12 @@ impl Parser {
         self.add_node(node);
     }
 
+    fn parse_comment_block(&mut self)  {
+        let _ = self.expect(TokenType::CommentStart);
+        let _ = self.next_token();
```

Comment:
can you run clippy? you'll need the nightly version of rust, do a `cargo update` and a `cargo build --features "dev"`.

You shouldn't need the `let _` bit